webapp/src/webusb.tsx

```diff
@@ -200,11 +200,17 @@ function showConnectionSuccessAsync(confirmAsync: ConfirmAsync, willTriggerDownl
 
 function showConnectionFailureAsync(confirmAsync: ConfirmAsync, showDownloadAsFileButton?: boolean) {
     const boardName = getBoardName();
+    const tryAgainText = lf("Try Again");
+    const helpText = lf("Help");
+    const downloadAsFileText = lf("Download as file");
```

Comment:
Was the intent to use `downloadAsFileText` and `helpText` elsewhere in the file? Seems like `downloadAsFileText` would need to go in `showPairStepAsync` which would require a broader scope, and I'm guessing that's why you opted not to use it there...but figured I'd check.

Reply:
Yeah, I just pulled them out so the lf on the rest of the description didn't get nested, not attempting to reduce redundancy there as it's a little messy (e.g. can't put it as top level as lf requires translations to be loaded )

Reply:
Do we have any styling rules for when/how to capitalize the button text? Seems a little inconsistent now with the `Download as file` capitalization, but maybe it's fine to use sentence case since it's a secondary button?

Reply:
we're a little inconsistent yeah; I had copied it form 
<img width="157" alt="image" src="https://user-images.githubusercontent.com/5615930/235780794-4f674bb2-87ce-4fe0-a994-3b976efb36e9.png">
but I'd say we typically lean more towards Title Casing buttons
<img width="167" alt="image" src="https://user-images.githubusercontent.com/5615930/235780873-e6276561-f23d-4422-9188-7cab52d0d3a0.png">
I'll swap these two to title case here as well